front/pages/poke/[wId]/conversations/[cId]/index.tsx

```diff
@@ -73,52 +74,56 @@ const ContentFragmentView = ({ message }: { message: ContentFragmentType }) => {
+  const { conversation } = useConversation({ workspaceId, conversationId });
+
   return (
     <div className="min-h-screen bg-structure-50">
       <PokeNavbar />
-      <div className="mx-auto max-w-4xl pt-8">
-        <Page.Vertical align="stretch">
-          {c.content.map((messages, i) => {
-            return (
-              <div key={`messages-${i}`}>
-                {messages.map((m, j) => {
-                  switch (m.type) {
-                    case "agent_message": {
-                      return (
-                        <AgentMessageView
-                          message={m}
-                          key={`message-${i}-${j}`}
-                        />
-                      );
-                    }
-                    case "user_message": {
-                      return (
-                        <UserMessageView
-                          message={m}
-                          key={`message-${i}-${j}`}
-                        />
-                      );
+      {conversation && (
+        <div className="mx-auto max-w-4xl pt-8">
+          <Page.Vertical align="stretch">
+            {conversation.content.map((messages, i) => {
+              return (
+                <div key={`messages-${i}`}>
+                  {messages.map((m, j) => {
+                    switch (m.type) {
+                      case "agent_message": {
+                        return (
+                          <AgentMessageView
+                            message={m}
+                            key={`message-${i}-${j}`}
+                          />
+                        );
+                      }
+                      case "user_message": {
+                        return (
+                          <UserMessageView
+                            message={m}
+                            key={`message-${i}-${j}`}
+                          />
+                        );
+                      }
+                      case "content_fragment": {
+                        return (
+                          <ContentFragmentView
+                            message={m}
+                            key={`message-${i}-${j}`}
+                          />
+                        );
+                      }
+                      default:
+                        assertNever(m);
                     }
-                    case "content_fragment": {
-                      return (
-                        <ContentFragmentView
-                          message={m}
-                          key={`message-${i}-${j}`}
-                        />
-                      );
-                    }
-                    default:
-                      assertNever(m);
-                  }
-                })}
-              </div>
-            );
-          })}
-        </Page.Vertical>
-      </div>
+                  })}
+                </div>
+              );
+            })}
+          </Page.Vertical>
+        </div>
+      )}
     </div>
   );
 };
```

Comment:
`nit` not need for default export anymore, React is moving away from this old "requirements".

Reply:
wdym by default export?

Reply:
Line 126 we have:
```
export default ConversationPage;
```

It was a pattern in React required by `React.lazy` for dynamic imports, as they simplify the syntax required for code-splitting and lazy loading of components. It was also used a lot for single component file. Javascript ecosystem evolved a lot and now they are generally bad because you can assign them any name when importing them.

Reply:
ack :+1: